### PR TITLE
US113634: Rubric feedback copy UI

### PR DESCRIPTION
**Pull request by lvyuan95-d2l**

### CODE REVIEW

File: editor/d2l-rubric-visibility-editor.js [hunk 1]
```diff
@@ -143,6 +155,14 @@ Polymer({
 					}
 				}
 			}
+
+			const feedbackCopyAction = entity && entity.getActionByName('enable-feedback-copy');
+			dom(this.root).querySelector('#enable-feedback-copy').hidden = feedbackCopyAction ? false : true;
```
Comment (rgoessl) [inline editor/d2l-rubric-visibility-editor.js:160]: Should only do query selection once, save result.

Comment (lvyuan95-d2l) [reply editor/d2l-rubric-visibility-editor.js:160]: You mean something like this?
```
const option = dom(this.root).querySelector('#enable-feedback-copy')
option.hidden = xxx
```

**Comment by lvyuan95-d2l** [comment]
> Looks good, just fix CI

The tests in CI have been failing for months somehow 🤔

**Comment by rgoessl** [comment]
> 
> 
> > Looks good, just fix CI
> 
> The tests in CI have been failing for months somehow 🤔

Just merge then

**Comment by mpharoah-d2l** [comment]
Yeah for some reason the CI always fails on the CI builds, but works in the builds triggered on merge
¯\\_(ツ)_/¯

**Pull request merged**